Update snapshot-restore.md (#8734)
opensearch-project/documentation-website: _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

   sudo ./bin/opensearch-keystore add s3.client.default.secret_key
   ```

> 1. (Optional) If you're using custom S3 endpoint (e.g. MinIO), disable AWS EC2 metadata connection:

Reviewer: ```suggestion
1. (Optional) If you're using a custom S3 endpoint (for example, MinIO), disable AWS EC2 metadata connection:
```

> 

Reviewer 2: Line 113: The service is named "Amazon EC2" (never "AWS EC2"). Additionally, if these are the first appearances of these service names, they should be "Amazon Simple Storage Service (Amazon S3) endpoint" and "Amazon Elastic Compute Cloud (Amazon EC2) metadata connection". If these are not the first appearances, "S3 endpoint" and "EC2 metadata connection" are fine.

   ```bash
   export AWS_EC2_METADATA_DISABLED=true
   ```

>    If you're installing OpenSearch via Helm, put this in your values file:

Reviewer: Does this 
```suggestion
   If you're installing OpenSearch using Helm, append the following to your values file:
```

Author: You're not appending, you're changing an existing parameter with an empty value (extraEnvs: []): https://artifacthub.io/packages/helm/opensearch-project-helm-charts/opensearch?modal=values

Reviewer: Got it. I updated the suggestion.
